### PR TITLE
Nerf rad goat again and minor rad anomaly tweaks

### PR DESCRIPTION
**Pull request by warface1234455**
Closes #16989
Also nerf rad goat because people whining about it
# Document the changes in your pull request
- Rad particles irradiation from rad anomaly is higher than standard rad particles (500 irradiation to 400 irradiation)
- Rad anomaly ball counts per shot 10->15
- Nerf rad goat healing from anomaly particles (10->1)
- Nerf rad goat health 200-75
- fix rad particles damaging rad goat

# Wiki Documentation
In doc of changes

# Changelog

:cl:  
tweak: Rad particles irradiation from rad anomaly is higher than standard rad particles by 100 (500 irradiation to 400 irradiation)
tweak: Rad anomaly ball counts per shot 10->15
tweak: Nerf rad goat healing from anomaly particles(10->1)
tweak: Nerf rad goat health 200->75
bugfix: fix rad particles damaging rad goat
/:cl:

### CODE REVIEW

File: code/modules/projectiles/projectile/energy/nuclear_particle.dm [hunk 1]
```diff
@@ -30,7 +30,7 @@
 	P.fire(angle)
 
 /obj/item/projectile/energy/nuclear_particle/wimpy
-	irradiate = 100
+	irradiate = 500
```
Comment (Moltijoe) [inline code/modules/projectiles/projectile/energy/nuclear_particle.dm]: no, radiation is the whole problem

File: yogstation/code/modules/mob/living/simple_animal/friendly/goats.dm [hunk 2]
```diff
@@ -182,13 +184,17 @@
...
+
 /mob/living/simple_animal/hostile/retaliate/goat/radioactive/on_hit(obj/item/projectile/P)
 	. = ..()
 	if(istype(P, /obj/item/projectile/energy/nuclear_particle))
 		// abosrbs nuclear particle to heal
-		P.damage = 0
-		adjustBruteLoss(-10)
-		adjustFireLoss(-10)
+		adjustBruteLoss(1)
+		adjustFireLoss(1)
```
Comment (Moltijoe) [inline yogstation/code/modules/mob/living/simple_animal/friendly/goats.dm]: don't forget the negative sign

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/projectile/energy/nuclear_particle.dm (hunk 1 above)
Comment (ShoziOffical) [reply code/modules/projectiles/projectile/energy/nuclear_particle.dm]: that is irradiation, not radiation pulse

Comment (ShoziOffical) [reply code/modules/projectiles/projectile/energy/nuclear_particle.dm]: it only affects mob not surrounding

File: yogstation/code/modules/mob/living/simple_animal/friendly/goats.dm (hunk 2 above)
Comment (ShoziOffical) [reply yogstation/code/modules/mob/living/simple_animal/friendly/goats.dm]: ah shit

File: code/modules/projectiles/projectile/energy/nuclear_particle.dm (hunk 1 above)
Comment (ShoziOffical) [reply code/modules/projectiles/projectile/energy/nuclear_particle.dm]: so if it hits an object, it only damaging it not spreading radiation contamination but if it hits a mob its radiation will increase

Comment (ShoziOffical) [reply code/modules/projectiles/projectile/energy/nuclear_particle.dm]: you know what im just gonna use the base nuclear

File: code/game/objects/effects/anomalies.dm [hunk 3]
```diff
@@ -364,12 +364,12 @@
...
 	density = TRUE
-	var/has_effect = TRUE //For goat spawning
+	var/spawn_goat = TRUE //For goat spawning
 
 /obj/effect/anomaly/radiation/anomalyEffect()
 	..()
-	for(var/i = 1 to 10)
-		fire_nuclear_particle_wimpy()
+	for(var/i = 1 to 15)
+		fire_nuclear_particle()
```
Comment (Moltijoe) [inline code/game/objects/effects/anomalies.dm:372]: not only making them the stronger particles, but firing 5 more?
that's a pretty significant increase

Comment (ShoziOffical) [reply code/game/objects/effects/anomalies.dm:372]: well i want the radiation anomaly to be lethal and not just a small orb shooting tickle particles that barely harming mob. 
I want the crew to actually avoid it, and force them to use a radiation suit when going near it as rad suit protects them from being harmed by rad particles

**Review by Moltijoe — APPROVED** [review]
i hesitantly approve this, i have the feeling there might be another PR after this though

**Comment by ShoziOffical** [comment]
> i hesitantly approve this, i have the feeling there might be another PR after this though

Well there will be, as i will make rad goat actually an antag

**Pull request merged**